docs/resources.md

```diff
@@ -194,7 +194,7 @@ spec:
 ```
 
 If no value is specified for `outputImageDir`, it will default to
-`/builder/image-outputs/{resource-name}`.
```

Comment:
hm why is this moving to `/builder/home`? (maybe it could be explained in the commit message?)

Reply:
@nader-ziada ^^

Reply:
from the commit message "move the default image output folder to /builder/home since this will always be mounted in the containers"

Reply:
> "move the default image output folder to /builder/home since this will always be mounted in the containers"

this is still a bit odd to me - as far as i can tell `/builder/home` is actually the home dir on the executing node? which means that we're expecting the output dir to be the home dir on the node 🤔 i dont want to draw this PR out any further tho so let's go with it for now - I also created #1030 to look further into the paths we're using